File: invoice-manager-models/src/main/java/com/business/unknow/model/cdfi/Concepto.java
package com.business.unknow.model.cdfi;

public class Concepto {

	private String ClaveProdServ;
	private String NoIdentificacion;
	private String ClaveUnidad;
	private String Unidad;
	private String Descripcion;
	private Double ValorUnitario;
	private Double Importe;
	private Double descuento;

	public String getClaveProdServ() {
		return ClaveProdServ;
	}

	public void setClaveProdServ(String claveProdServ) {
		ClaveProdServ = claveProdServ;
	}

	public String getNoIdentificacion() {
		return NoIdentificacion;
	}

	public void setNoIdentificacion(String noIdentificacion) {
		NoIdentificacion = noIdentificacion;
	}

	public String getClaveUnidad() {
		return ClaveUnidad;
	}

	public void setClaveUnidad(String claveUnidad) {
		ClaveUnidad = claveUnidad;
	}

	public String getUnidad() {
		return Unidad;
	}

	public void setUnidad(String unidad) {
		Unidad = unidad;
	}

	public String getDescripcion() {
		return Descripcion;
	}

	public void setDescripcion(String descripcion) {
		Descripcion = descripcion;
	}

	public Double getValorUnitario() {
		return ValorUnitario;
	}

	public void setValorUnitario(Double valorUnitario) {
		ValorUnitario = valorUnitario;
	}

	public Double getImporte() {
		return Importe;
	}

	public void setImporte(Double importe) {
		Importe = importe;
	}

	public Double getDescuento() {
		return descuento;
	}

	public void setDescuento(Double descuento) {
		this.descuento = descuento;
	}
}

File: invoice-manager-ui/src/app/models/factura/concepto.ts
import { Impuesto } from './impuesto';

export class Concepto{
    public id : number;
	public claveProdServ : string;
	public noIdentificacion : string;
	public cantidad : number;
	public claveUnidad : string;
	public descripcionCUPS: string;
	public unidad : string;
	public descripcion : string;
	public valorUnitario : number;
	public importe : number;
	public descuento : number;
	public retencion : number;
	public impuestos : Impuesto[];
	public retenciones : Impuesto[];

	//fields only used on UI
	public iva: boolean;
	public retencionFlag: boolean;
	
	constructor(){
		this.descuento = 0;
		this.cantidad = 0;
		this.valorUnitario = 0;
		this.importe = 0;
		this.retencion=0;
		this.impuestos = [];
		this.retenciones = [];
		this.iva = true;
		this.retencionFlag=false;
	}

	
}